### PR TITLE
adding minimal support

### PR DESCRIPTION
**Pull request by parithosh**

### CODE REVIEW

File: config-example/cl/config.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # Extends the mainnet preset
-PRESET_BASE: 'mainnet'
+PRESET_BASE: '$PRESET_BASE'
```
Comment (barnabasbusa) [inline config-example/cl/config.yaml:2]: I don't think you need the quotes on here anymore

**Comment by barnabasbusa** [comment]
```sh
Caused by: tech.pegasys.teku.infrastructure.ssz.sos.SszDeserializeException: First variable element offset doesn't match the end of fixed part
at tech.pegasys.teku.infrastructure.ssz.schema.impl.AbstractSszContainerSchema.sszDeserializeTree(AbstractSszContainerSchema.java:303) ~[teku-infrastructure-ssz-24.3.0.jar:24.3.0]
at tech.pegasys.teku.infrastructure.ssz.schema.SszSchema.sszDeserialize(SszSchema.java:80) ~[teku-infrastructure-ssz-24.3.0.jar:24.3.0]
at tech.pegasys.teku.infrastructure.ssz.schema.SszSchema.sszDeserialize(SszSchema.java:84) ~[teku-infrastructure-ssz-24.3.0.jar:24.3.0]
at tech.pegasys.teku.spec.Spec.deserializeBeaconState(Spec.java:317) ~[teku-ethereum-spec-24.3.0.jar:24.3.0]
at tech.pegasys.teku.spec.datastructures.util.ChainDataLoader.loadState(ChainDataLoader.java:24) ~[teku-ethereum-spec-24.3.0.jar:24.3.0]
at tech.pegasys.teku.services.beaconchain.WeakSubjectivityInitializer.getAnchorPoint(WeakSubjectivityInitializer.java:88) ~[teku-services-beaconchain-24.3.0.jar:24.3.0]
at tech.pegasys.teku.services.beaconchain.WeakSubjectivityInitializer.lambda$loadInitialAnchorPoint$0(WeakSubjectivityInitializer.java:57) ~[teku-services-beaconchain-24.3.0.jar:24.3.0]
```

**Comment by parithosh** [comment]
can we track this in the internal ethpandaops repo? https://github.com/ethpandaops/ethPandaOps/issues/1117

**Review by Eianaftyio — COMMENTED** [review]
.diff

**Review by Eianaftyio — COMMENTED** [review]
.diff

**Comment by Eianaftyio** [comment]
.patch

**Comment by Eianaftyio** [comment]
.diff

**Review by Eianaftyio — COMMENTED** [review]
.diff

**Review by Eianaftyio — COMMENTED** [review]
.diff

**Review by Eianaftyio — COMMENTED** [review]
.diff.patch

File: config-example/cl/config.yaml (hunk 1 above)
Comment (barnabasbusa) [inline config-example/cl/config.yaml:2]:
```suggestion
PRESET_BASE: $PRESET_BASE
```

**Review by barnabasbusa — APPROVED** [review]

**Pull request merged**